webpack.config.js

```diff
@@ -42,9 +42,13 @@ const BABEL_CONFIG = {
 };
 
 const CSS_CONFIG = {
-  localIdentName: devMode
-    ? "[name]__[local]___[hash:base64:5]"
-    : "[hash:base64:5]",
+  modules: {
+    auto: filename => !filename.includes("node_modules"),
```

Comment:
Seems we're still relying on loading style from ACE editor

#### `master`
![Screenshot 2024-02-23 at 4 10 03 PM](https://github.com/metabase/metabase/assets/1937582/86311ecf-9a37-45f8-bc5c-ad2b8bb777fa)


#### This PR
![Screenshot 2024-02-23 at 4 03 35 PM](https://github.com/metabase/metabase/assets/1937582/bf30875d-5edb-47cd-9b42-36fed493d08c)

I checked that the rules for ace classes like `.ace_text-layer` is missing.